src/integration-test/java/com/commercetools/sync/integration/externalsource/products/ProductSyncIT.java

```diff
@@ -129,8 +129,12 @@ private ProductSyncOptions buildSyncOptions() {
         final BiConsumer<String, Throwable> errorCallBack = (errorMessage, exception) -> {
             errorCallBackMessages.add(errorMessage);
             errorCallBackExceptions.add(exception);
+            System.err.print(errorMessage);
+        };
+        final Consumer<String> warningCallBack = warningMessage -> {
+            warningCallBackMessages.add(warningMessage);
+            System.out.print(warningMessage);
```

Comment:
I know that it is just an example and within test usage of System.out is ok but for actual code we should have checkstyle rule which will automatically break the build.

Reply:
@butenkor Actually this line was already removed from this PR because PMD broke the build because of it 😄 . This change will be made later in next release. So this PR only includes only .md file changes..